modular_skyrat/master_files/code/datums/components/crafting.dm

```diff
@@ -0,0 +1,4 @@
+/datum/mind/proc/unteach_crafting_recipe(recipe)
```

Comment:
A psychic you might have been, but you forgot this one :)

Reply:
fuuuu-